src/playwright-runner.js

```diff
@@ -286,7 +286,20 @@ async function run (nodeBin, runCfgPath, suiteName) {
 
   const excludeParams = ['screenshot-on-failure', 'video', 'slow-mo'];
 
+  let usePlaywrightCfg = false;
+  const defaultCfgFiles = ['./playwright.config.ts', './playwright.config.js'];
+  for (const file of defaultCfgFiles) {
+    if (fs.existsSync(path.join(projectPath, file)) === true) {
+      usePlaywrightCfg = true;
+    }
+  }
   for (let [key, value] of Object.entries(args)) {
+    // There is a conflict when playwright project and `browser` both existing.
```

Comment:
Just explaining the conflict a bit more in depth.

```suggestion
    // There is a conflict if the playwright project has a `browser` defined, since the job is launched with the browser set by saucectl, which is now set as the job's metadata.
```

Reply:
sure thing